bin/p-s.js

```diff
@@ -46,7 +47,7 @@ runPackageScript({
 }, function(result) {
   if (result.error) {
     log.error(result.error)
-    const FAIL_CODE = 1
```

Comment:
This file isn't compiled so it has to be Node 0.10.0 friendly. Const is only available behind a flag in Node 0.10.0 [node.green](http://node.green/#const)